### ISSUE TITLE
Pronto crash on Travis CI

### ISSUE THREAD

**Issue by Schweigert** [prontolabs/pronto#277]
Pronto is crashing on Travis CI :/

```bash
 $ export PULL_REQUEST_ID=${TRAVIS_PULL_REQUEST} && bundle exec pronto run -f github_pr -c origin/${TRAVIS_BRANCH}
/home/travis/.rvm/gems/ruby-2.4.1/gems/pronto-0.9.5/lib/pronto/git/repository.rb:52:in `new': object not found - no match for id (0aa3f72db9c820b0428dffe1785eb471fe06b6f1) (Rugged::OdbError)
	from /home/travis/.rvm/gems/ruby-2.4.1/gems/pronto-0.9.5/lib/pronto/git/repository.rb:52:in `blame'
	from /home/travis/.rvm/gems/ruby-2.4.1/gems/pronto-0.9.5/lib/pronto/git/patch.rb:17:in `blame'
	from /home/travis/.rvm/gems/ruby-2.4.1/gems/pronto-0.9.5/lib/pronto/git/line.rb:45:in `blame'
	from /home/travis/.rvm/gems/ruby-2.4.1/gems/pronto-0.9.5/lib/pronto/git/line.rb:20:in `commit_sha'
	from /home/travis/.rvm/gems/ruby-2.4.1/gems/pronto-0.9.5/lib/pronto/message.rb:18:in `initialize'
	from /home/travis/.rvm/gems/ruby-2.4.1/gems/pronto-flay-0.9.0/lib/pronto/flay.rb:48:in `new'
	from /home/travis/.rvm/gems/ruby-2.4.1/gems/pronto-flay-0.9.0/lib/pronto/flay.rb:48:in `new_message'
	from /home/travis/.rvm/gems/ruby-2.4.1/gems/pronto-flay-0.9.0/lib/pronto/flay.rb:35:in `block in messages'
	from /home/travis/.rvm/gems/ruby-2.4.1/gems/pronto-flay-0.9.0/lib/pronto/flay.rb:28:in `map'
	from /home/travis/.rvm/gems/ruby-2.4.1/gems/pronto-flay-0.9.0/lib/pronto/flay.rb:28:in `messages'
	from /home/travis/.rvm/gems/ruby-2.4.1/gems/pronto-flay-0.9.0/lib/pronto/flay.rb:9:in `run'
	from /home/travis/.rvm/gems/ruby-2.4.1/gems/pronto-0.9.5/lib/pronto/runners.rb:20:in `block in run'
	from /home/travis/.rvm/gems/ruby-2.4.1/gems/pronto-0.9.5/lib/pronto/runners.rb:13:in `each'
	from /home/travis/.rvm/gems/ruby-2.4.1/gems/pronto-0.9.5/lib/pronto/runners.rb:13:in `run'
	from /home/travis/.rvm/gems/ruby-2.4.1/gems/pronto-0.9.5/lib/pronto.rb:64:in `run'
	from /home/travis/.rvm/gems/ruby-2.4.1/gems/pronto-0.9.5/lib/pronto/cli.rb:61:in `block in run'
	from /home/travis/.rvm/gems/ruby-2.4.1/gems/pronto-0.9.5/lib/pronto/cli.rb:60:in `chdir'
	from /home/travis/.rvm/gems/ruby-2.4.1/gems/pronto-0.9.5/lib/pronto/cli.rb:60:in `run'
	from /home/travis/.rvm/gems/ruby-2.4.1/gems/thor-0.19.4/lib/thor/command.rb:27:in `run'
	from /home/travis/.rvm/gems/ruby-2.4.1/gems/thor-0.19.4/lib/thor/invocation.rb:126:in `invoke_command'
	from /home/travis/.rvm/gems/ruby-2.4.1/gems/thor-0.19.4/lib/thor.rb:369:in `dispatch'
	from /home/travis/.rvm/gems/ruby-2.4.1/gems/thor-0.19.4/lib/thor/base.rb:444:in `start'
	from /home/travis/.rvm/gems/ruby-2.4.1/gems/pronto-0.9.5/bin/pronto:6:in `<top (required)>'
	from /home/travis/.rvm/gems/ruby-2.4.1/bin/pronto:23:in `load'
	from /home/travis/.rvm/gems/ruby-2.4.1/bin/pronto:23:in `<main>'
	from /home/travis/.rvm/gems/ruby-2.4.1/bin/ruby_executable_hooks:15:in `eval'
	from /home/travis/.rvm/gems/ruby-2.4.1/bin/ruby_executable_hooks:15:in `<main>'
The command "export PULL_REQUEST_ID=${TRAVIS_PULL_REQUEST} && bundle exec pronto run -f github_pr -c origin/${TRAVIS_BRANCH}" exited with 1.```

This is the build page:https://travis-ci.org/KrakenLab/kraken/builds/323167677?utm_source=github_status&utm_medium=notification

**Comment by mknapik** [comment]
Not sure if `PULL_REQUEST_ID` is still supported. Could you try using `PRONTO_PULL_REQUEST_ID` instead?

**Comment by doomspork** [comment]
It should throw a deprecation warning but I don't see that here 🤔  https://github.com/prontolabs/pronto/blob/669177ddbda2c2f6502b9691188e032a0127fbdc/lib/pronto/client.rb#L11

**Comment by Schweigert** [comment]
Yes. I change to PRONTO_PULL_REQUEST_ID :+1: 
It's breaking with Ruby 2.6dev (Travis ruby-head version) now. :/

**Comment by npetkov** [comment]
@Schweigert Try using [rugged](https://github.com/libgit2/rugged) from master, it's fixed for 2.5/2.6 but still no new release though.

**Comment by Schweigert** [comment]
Thanks @npetkov  =D
